localstripe/resources.py

```diff
@@ -4017,7 +4017,7 @@ class IssuingDispute(StripeObject):
     _id_prefix = 'idp_'
     _id_length = 24
 
-    def __init__(self, transaction: str, currency: str = 'usd', metadata: dict = None, evidence: dict = None):
+    def __init__(self, transaction: str, currency: str = 'usd', metadata: dict = None, evidence: dict = None, amount = None):
```

Comment:
Do we really need validation here and are we always sending an amount (unclear to me since we removed this field at some point)? Also, we're not doing anything with this value. Should we save it on the IssuingDispute object?  I'm trying to strike a balance where localstripe isn't unnecessarily brittle ..

Reply:
I think at this point we are sending it. But it is an optional field, so we dont have to then